charts/flyte-binary/templates/configmap.yaml

```diff
@@ -26,6 +26,9 @@ data:
       {{- end }}
       endpoint: localhost:8089
       insecure: true
+      {{- if .Values.configuration.auth.internal.audience }}
+      audience: {{ .Values.configuration.auth.internal.audience }}
+      {{- end}}
```

Comment:
```suggestion
      {{- if .Values.configuration.auth.internal.audience }}
      audience: {{ .Values.configuration.auth.internal.audience }}
      {{- end}}
      {{- end }}
      endpoint: localhost:8089
      insecure: true
```

Reply:
Would it be reasonable to keep it alongside the `clientId` parameter, and activate only when `auth` is enabled?

Reply:
@jeevb Yeah just added in your suggestion to the chart. Should be good to go for the merge?